webex-linux/webex.sh

```diff
@@ -30,3 +32,5 @@ webex() {
 		fi
 	fi;
 }
+
+webex
```

Comment:
Can you remove this call? The code was intended to be sourced e.g. in your .bashrc. Afterwards you are able to just run `webex` in your terminal.